understanding/22/redundant-entry.html

```diff
@@ -117,6 +117,16 @@ <h3>Failures for Redundant Entry</h3>
       </section>
       
    </section>
+
+         <section id="resources">
+        <h2>Resources</h2>
+
+        <ul>
+            <li><a href="https://rawgit.com/w3c/coga/master/gap-analysis/#table3">Cognitive Accessibility Gap Analysis Topic 3: Entering Data, Error Prevention, &amp; Recovery</a></li>
```

Comment:
link to https://www.w3.org/TR/coga-gap-analysis/#table3 or https://w3c.github.io/coga/gap-analysis/#table3